tests/assets/fixtures/datasets.py

```
@@ -0,0 +1,56 @@
+"""Dataset fixtures for unit testing."""
+
+from pathlib import Path
+from omegaconf import OmegaConf
```

Ruff (0.8.2)

`omegaconf.OmegaConf` imported but unused

Remove unused import: `omegaconf.OmegaConf`

(F401)